.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -7,18 +7,21 @@ assignees: ''
 
 ---
 
+## Open Release
+
 ### TASKS:
 
+ - [ ] [DOC] Ping in #documentation-internal about the new release
  - [ ] [GIT] Create branch release/X.X.X in owncloud/android from master
  - [ ] [DEV] Update version number and name in build.gradle in owncloudApp module
- - [ ] [DIS] Create a folder for the new version like $majorVersion.$minorVersion.$patchVersion_YYYY-MM-DD inside the `changelog` folder
+ - [ ] [DIS] Create a folder for the new version like `X.X.X_YYYY-MM-DD` inside the `changelog` folder
  - [ ] [DIS] Move all changelog files from the unreleased folder to the new version folder
  - [ ] [DIS] Update screenshots, if needed, in README.md
  - [ ] [DIS] Add ReleaseNotes replacing `emptyList` with `listOf` and adding inside `ReleaseNote()` with String resources
  - [ ] [QA] Design Test plan
  - [ ] [QA] Regression Test plan
- - [ ] [GIT] Create and sign tag 'oc-android-X.X.X' in HEAD commit of release branch, in owncloud/android
- - [ ] [GIT] Create and sign tag 'Y.Y.Y' in HEAD commit of release branch, in owncloud/android-library
+ - [ ] [GIT] Create and sign tag `oc-android-X.X.X` in HEAD commit of release branch, in owncloud/android
+ - [ ] [GIT] Create and sign tag `Y.Y.Y` in HEAD commit of release branch, in owncloud/android-library
```

Comment:
Maybe it's better to represent major version, minor version and patch version numbers with different letters or variables? It could lead to confusion maybe thinking it is the same number the three times

Reply:
i used the same letters as in all document. X -> app, Y -> library. X.Y.Z may lead to confusion... change in the whole document could be an option.

Reply:
I'll use the following

X.Y.Z (uppercase) -> app
x.y.z (lowercase) -> library

Reply:
I would do change it to avoid confusion. `M.m.p` as @abelgardep suggested seems a good option!

Reply:
ok, will go that direction!

Reply:
done!